Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM node:22.17.1-alpine
 WORKDIR /app
 COPY .output ./
+COPY data ./data
```

Comment:
@alifarooq93 This should already be fixed in the current main. I think you have merge conflicts...